aws-api/src/main/java/com/amplifyframework/api/aws/GsonGraphQLResponseFactory.java

```diff
@@ -18,6 +18,9 @@
 import com.amplifyframework.AmplifyException;
 import com.amplifyframework.api.ApiException;
 import com.amplifyframework.api.graphql.GraphQLResponse;
+import com.amplifyframework.core.model.scalar.AWSDate;
```

Comment:
Makes sense to me.  Initially I had separated them since they are AppSync specific models; however, given that they are in `core`, they can and will be used more broadly as AWS models of time.